function.js

```diff
@@ -0,0 +1,145 @@
+// // Перепишите функцию, чтобы она делала то же самое, но без if, в одну строку.
+// // Сделайте два варианта функции checkAge:
+// // Используя оператор ?
+// // Используя оператор ||
+
+function checkAge(age) {
+    return (age >= 18) ? true : confirm("Родители разрешили?")
+};
+
+function checkAge(age) {
+    return (age >= 18) || confirm("Родители разрешили?")
+};
+
+function checkAge(age) ? () => true: () confirm("Родители разрешили?");
+
+
+// // Напишите функцию min(a,b), которая возвращает меньшее из чисел a и b.
+function getMin(a, b) {
+    return (a < b) ? a : b
+};
+
+// Напишите функцию, которая принимает 2 числа и возвращает 1, если первое число больше, чем второе; -1, если первое число меньше, чем второе, и 0, если числа равны.
+
+function getVar(a, b) {
+    if (a > b) {
+        return 1;
+    } else(b > a) {
+        return -1;
+    };
+}
+console.log(getVar(2, 3));
+
+let getVar = (a, b) => {
+    return (a > b) ? 1 : (b > a) ? -1
```

Comment:
the task is not completed

Reply:
wrong syntax

do you check tasks?

you can't use ternary operator without ? or :

Reply:
edited